Makefile

```diff
@@ -186,7 +186,7 @@ generate: app/controllers.go assets/js/client.js bindata_assetfs.go migration/sq
 .PHONY: dev
 dev: prebuild-check deps generate $(FRESH_BIN)
 	docker-compose up -d db
-	$(FRESH_BIN)
```

Comment:
should remove the mention of setting in readme.adoc for "make dev" too if this is always going to be set.